src/lib/core/overlay/overlay-ref.ts

```diff
@@ -35,10 +35,15 @@ export class OverlayRef implements PortalHost {
     }
 
     let attachResult = this._portalHost.attach(portal);
+
+    // Update the pane element with the given state configuration.
     this.updateSize();
     this.updateDirection();
     this.updatePosition();
 
+    // Enable pointer events for the overlay pane element.
+    this._togglePointerEvents(true);
```

Comment:
Yeah they are enabled by default. But developers can always call `attach` after `detach` has been called. This is necessary to revert the `togglePointerEvents(false)` from the detach method.